### PR TITLE
Explain the process of releasing to the app stores

### PR DESCRIPTION
**Pull request by andybalaam**
### Pull Request Checklist

- [ ] Changes has been tested on an Android device or Android emulator with API 21
- [ ] UI change has been tested on both light and dark themes
- [ ] Accessibility has been taken into account. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#accessibility
- [x] Pull request is based on the develop branch
- [ ] Pull request includes a new file under ./changelog.d. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#changelog
- [ ] Pull request includes screenshots or videos if containing UI changes
- [ ] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
- [x] You've made a self review of your PR
- [ ] If you have modified the screen flow, or added new screens to the application, you have updated the test [UiAllScreensSanityTest.allScreensTest()](https://github.com/vector-im/element-android/blob/main/vector/src/androidTest/java/im/vector/app/ui/UiAllScreensSanityTest.kt#L73)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,20 @@ At each Element release, the SDK module is copied to a dedicated repository: htt
...
+* Each release on the Google Play Store undergoes review by Google before it comes out. This can take an unpredictable amount of time. In some cases it has taken several weeks.
+* In order for F-Droid to guarantee that the app you receive exactly matches the public source code, they build releases themselves. When a release is considered stable, Element staff inform the F-Droid maintainers and it is added to the build queue. Depending on the load on F-Droid's infrastructure, it can take some time for releases to be built. This always takes at least 24 hours, and can take several days.
+
+If you would like to receive releases more quickly (bearing in mind that they may not be stable) you have a number of options:
+
+1. [Sign up to receive beta releases](https://play.google.com/apps/testing/im.vector.app) via the Google Play Store.
+2. Install a [release APK](https://github.com/vector-im/element-android/releases) directly - download the relevant .apk file and allow installing from untrusted sources in your device settings.
```
Comment (Ornanovitch) [inline README.md:41]: could it be worth to precise that GH builds are only ment for GPlay variant? Or is this a wrong assumption?

Thanks for this anyway, really helpful :)

Comment (andybalaam) [reply README.md:41]: Ah yes definitely.  @bmarty is that correct?

Comment (bmarty) [reply README.md:41]: No, F-Droid debug is also build by GitHub action as per https://github.com/vector-im/element-android/blob/develop/.github/workflows/build.yml#L22. I do not know where are exported the APKs though

Comment (bmarty) [reply README.md:41]: Yes, so far release versions of F-Droid variant are not build by us, just debug versions.

Comment (bmarty) [reply README.md:41]: (I should have read more carefully the text)

**Review by bmarty — APPROVED** [review]
LGTM.

File: README.md (hunk 1 above)
Comment (andybalaam) [reply README.md:41]: @Ornanovitch I made a new PR with a small update to mention what you suggested, thanks!

**Pull request merged**